Templates/FaceLocator-NodeJS/index.js

```diff
@@ -0,0 +1,58 @@
+// Setup
+// 1) Go to https://www.microsoft.com/cognitive-services/en-us/computer-vision-api 
+//    Sign up for computer vision api
+// 2) Go to Function app settings -> App Service settings -> Settings -> Application settings
+//    create a new app setting Vision_API_Subscription_Key and use Computer vision key as value
+// 3) Go to Function app settings -> App Service settings -> Tools -> Console
+//    Enter the following commands: 
+//    > cd <functionName>
+//    > npm install
+module.exports = function (context, image) {
+    context.log("Image Size:", image.length);
+
+    var request = require('request-promise');
```

Comment:
make all requires at the top of file (preferably) or if that's not possible with functions, at the top of the function